products/facilities/facdb/cli.py

```diff
@@ -92,35 +87,13 @@ def run(
         None, "--all", help="Execute all datasets, both python and sql"
     ),
 ):
-    """
-    This function is used to execute the python portion of a pipeline,
-    if there's a scripts section in datasets.yml under this dataset, the
-    sql script will also be executed.
-    facdb run -n {{ name }} to run both python and sql part\n
-    facdb run -n {{ name }} --python to run the python part only\n
-    facdb run -n {{ name }} --sql to run the sql part only\n
-    facdb run --all\n
-    """
+    """ """
     datasets = read_datasets_yml()
-    dataset_names = (
```

Comment:
Formerly, this would search for a "script" (ie a python pipeline) and sql file, and conditionally run them. Now, we take care of the pipeline part with recipes, and just have to run the accompany sql files. This should be simplified - we should just specify to run all sql files in a given folder, as order doesn't appear to matter.